docs/user-guide/migration.md

```diff
@@ -0,0 +1,131 @@
+  - extensions/v1beta1
+  - networking.k8s.io/v1beta1
+
+    (You would get a message about deprecation but the ingress resource would get created.)
+
+From K8s version 1.22 onwards, you can ONLY set the "apiVersion:" field of a ingress resource, to the value "networking.k8s.io/v1". The reason is [official blog on deprecated ingress api versions](https://kubernetes.io/blog/2021/07/26/update-with-ingress-nginx/).
+
+(Reason #2) When you upgrade to K8s version v1.22 while you are already using the Ingress-Nginx-Controller, there are several scenarios where the old ingress objects will not work. Read this FAQ to check which scenario matches your use case.
+
+## What is ingressClassName field ?
+
+IngressClassName is the name of the IngressClass cluster resource. The associated IngressClass defines which controller will implement the resource. This replaces the deprecated `kubernetes.io/ingress.class` annotation. For backwards compatibility, when that annotation is set, it must be given precedence over this field. The controller may emit a warning if the field and annotation have different values. Implementations of this API should ignore Ingresses without a class specified. An IngressClass resource may be marked as default, which can be used to set a default value for this field.
+
+
+## I have only one ingress object in my cluster. What should I do ?
+
+- If you have only one ingress controller running in your environment and you still want to use ingress class, you should add the annotation "ingressclass.kubernetes.io/is-default-class" in your ingress class, so new Ingress objects will have this one as default ingressClass.
+
+In this case, you need to make your Controller aware of the objects. If you have several Ingress objects and they don't yet have the [ingressClassName](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#ingress-v1-networking-k8s-io) attribute, or the ingress annotation (`kubernetes.io/ingress.class`),  then you should start your ingress-controller with the flag `--watch-ingress-without-class=true"`.
+
+You can configure your helm chart installation's values file with `.controller.watchIngressWithoutClass: true`. 
+
+We highly recommend that you create the ingressClass as shown below:
+```
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+  name: nginx
+  annotations:
+    ingressclass.kubernetes.io/is-default-class: "true"
+spec:
+  controller: k8s.io/ingress-nginx
+```
+And add the value "spec.ingressClassName=nginx" in your Ingress objects
+
+## I have multiple ingress objects in my cluster. What should I do ?
+- If you don't care about ingress class, or you have a lot of Ingress objects without ingress class configuration, you can run the ingress-controller with the flag '--watch-ingress-without-class=true'.
```

Comment:
The answer should be “It depends” if they have only one ingress controller then they can start with nginx ingress controller with --watch-ingress-class-without resource or they should create a default class as you have mentioned in previous questions

You have already mentioned “If you don’t care about ingress class”, But If I was on the other end reading this doc, I would like to have some more details in the question itself. I mean it is pretty easy to understand what this statement mean but sometimes people will not spend a minute to think and move away without grasping the real deal.